UnityProject/Assets/Scripts/PlayGroups/PlayerScript.cs

```diff
@@ -12,7 +12,7 @@ public class PlayerScript : ManagedNetworkBehaviour
         // the maximum distance the player needs to be to an object to interact with it
 		//1.75 is the optimal distance to now have any direction click too far
 		//NOTE FOR ANYONE EDITING THIS IN THE FUTURE: Character's head is slightly below the top of the tile
-		//hence top reach is slightly lowered than bottom reach, where the legs go exactly to the bottom of the tile.
+		//hence top reach is slightly lower than bottom reach, where the legs go exactly to the bottom of the tile.
         public const float interactionDistance = 1.75f;

```

Comment:
In the future this should be calculated from the vector3int of the tile(s) the player is on, that would prevent the player sprite being of influence, but for now this is still a great find (yes not part of this PR, but great research!)